app/src/main/java/com/dscout/membranevideoroomdemo/viewmodels/RoomViewModel.kt

```diff
@@ -372,31 +359,21 @@ class RoomViewModel(
     }
 
     override fun onTrackUpdated(ctx: TrackContext) {
-        val p = mutableParticipants[ctx.peer.id]
         // Updates metadata of given track
-        if (ctx.metadata["type"] == "camera") {
-            if (p != null) {
-                mutableParticipants[ctx.peer.id] = p.copy(
-                    tracksMetadata = p.tracksMetadata + (
-                        (
-                            mutableParticipants[ctx.peer.id]?.videoTrack?.id()
-                                ?: ""
-                            ) to ctx.metadata
-                        )
+        if (mutableParticipants[ctx.peer.id] != null) {
```

Comment:
we can extract `mutableParticipants[ctx.peer.id]` as a new variable and that way it'll be shorter and we'll avoid `!!` operator

Reply:
Done.